roles/st2mistral/handlers/main.yml

```diff
@@ -1,3 +1,11 @@
+- name: enable mistral upgrade
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - /etc/mistral/mistral-db-manage.upgrade.head.ansible.has.run
+    - /etc/mistral/mistral-db-manage.populate.ansible.has.run
```

Comment:
Previous version looks good to me, just adding `skip_ansible_lint` tag to the task will make the CI happy. Ansible linter is being dumb here.